### ISSUE TITLE
No type check on document for full screen

### ISSUE THREAD

**Issue by ianrichard** [souporserious/react-media-player#31]
Webpack server won't start when using [request-fullscreen](https://github.com/souporserious/react-media-player/blob/master/src/utils/request-fullscreen.js) and [exit-fullscreen](https://github.com/souporserious/react-media-player/blob/master/src/utils/exit-fullscreen.js) because it refers to document, which doesn't exist without a browser.

```
return document.documentElement[curr] ? curr : prev;
           ^
ReferenceError: document is not defined
```

Fix: 

Wrap return with 

```js
if (typeof document !== 'undefined') {
...
}
```

Great library BTW!  Nice functionality and I love how decoupled it is.

**Comment by trda** [comment]
I have changed it, but still I am getting the same error.

**Comment by romanenko** [comment]
This should be fixed with PR #40. We should, probably, close this issue :)
